x/group/server/invariants.go

```diff
@@ -169,3 +181,125 @@ func groupTotalWeightInvariant(ctx sdk.Context, groupTable orm.Table, groupMembe
 	}
 	return msg, broken, err
 }
+
+func tallyVotesSumInvariant(ctx sdk.Context, proposalTable orm.AutoUInt64Table, groupAccountTable orm.PrimaryKeyTable, groupMemberByGroupIndex orm.UInt64Index, voteTable orm.PrimaryKeyTable) (string, bool, error) {
+	var msg string
+	var broken bool
+
+	var proposal group.Proposal
+	var groupAcc group.GroupAccountInfo
+	var groupMem group.GroupMember
+	var vote group.Vote
+
+	proposalIt, err := proposalTable.PrefixScan(ctx, 1, math.MaxUint64)
+	if err != nil {
+		return msg, broken, err
+	}
+	defer proposalIt.Close()
+
+	totalVotingWeight := apd.New(0, 0)
+	yesVoteWeight := apd.New(0, 0)
+	noVoteWeight := apd.New(0, 0)
+	abstainVoteWeight := apd.New(0, 0)
+	vetoVoteWeight := apd.New(0, 0)
```

Comment:
you need to reset them to 0 after each proposal

Reply:
Yeah, I forgot. Sorry for that
Thanks @blushi

Reply:
better to move these declarations at the beginning of the loop and then you don't need to reset them at the end of the loop

Reply:
okay